new-integration-tests/src/test/java/oracle/weblogic/kubernetes/TestConstants.java

```diff
@@ -20,7 +20,7 @@
   public static final String MANAGED_SERVER_NAME_BASE = "managed-server";
   public static final String WLS_DOMAIN_TYPE = "WLS";
   public static final String WLS_DEFAULT_CHANNEL_NAME = "default";
-  public static final String DEFAULT_WLS_IMAGE_TAGS = "12.2.1.3, 14.1.1.0";
+  public static final String DEFAULT_WLS_IMAGE_TAGS = "12.2.1.3, 14.1.1.0-11";
```

Comment:
Should we not verify 14.1.1.0 ?

Reply:
Did you see my earlier comment? tags changed in OCR for 1411 images.

Reply:
Should we verify 14.1.1.0 or not ? Let us bring this up in today scrum